src/pages/SNSPage.jsx

```diff
@@ -0,0 +1,53 @@
+import { css } from '@emotion/react';
+import { useEffect, useState } from 'react';
+
+import Header from '../components/header/Header.jsx';
+import Content from '../components/content/Content.jsx';
+import Footer from '../components/footer/Footer.jsx';
+import mockData from '../mock/mockData.json';
+
+const pageWrapper = css`
+  position: relative;
+  margin: 0 auto;
+  /* overflow: hidden; */
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 430px;
+  border: 1px solid var(--border-color);
+`;
+
+const otherNickName = '손오공';
+const otherName = 'svt';
+const currentName = 'carat';
```

Comment:
상수값은 `UPPER_SNAKE_CASE`로 작성해 주시면 됩니다!
```suggestion
const OTHER_NICKNAME = '손오공';
const OTHER_NAME = 'svt';
const CURRENT_NAME = 'carat';
```